samples/java-customer-registry-views-quickstart/src/main/proto/customer/api/customer_api.proto

```diff
@@ -46,16 +47,16 @@ message ChangeAddressRequest {
 }
 
 service CustomerService {
-  option (akkaserverless.service) = {
```

Comment:
I think I missed this in previous discussions: Do we not need the non-codegen annotation saying that it is a service, what entity type, etc - does the proxy now know of the codegen annotations, or did it never use the annotation metadata for anything proxy-side?

Reply:
The proxy is not aware of this annotation. The proxy get to know what kind of service and entity it is through the registration. That's also partially the goal of this PR, by changing some of the samples and having them running on CI we confirm that the proxy doesn't care about this annotation.

Reply:
Thanks for confirming!